commcare-cloud-bootstrap/bootstrap.sh

```diff
@@ -6,7 +6,7 @@ SPEC=$3
 
 commcare-cloud-bootstrap provision $SPEC --env $ENV
 while
-    commcare-cloud $ENV run-shell-command all 'echo {{ inventory_hostname }}' -u ubuntu --use-pem
+    commcare-cloud $ENV run-shell-command all 'echo {{ inventory_hostname }}' -u ubuntu --use-factory-auth
```

Comment:
I think this would be a great use case for the shiny new `ping` function :)

Also, `-u ubuntu` should now be unnecessary, since that's now included in `--use-factory-auth` (right?)

Reply:
Ended up changing it to default to root if `--use-factory-auth` is specified, here:
https://github.com/dimagi/commcare-cloud/pull/1702/commits/6ca956609be7f17cff3c83fdf32ad333e3895c3c